src/frontend/src/binder/expr/function.rs

```diff
@@ -922,6 +923,13 @@ impl Binder {
             ))
             .into());
         }
+        if matches!(self.context.clause, Some(Clause::GeneratedColumn)) {
```

Comment:
Can we also use `clause` to get rid of `BindFor::Ddl`?

Reply:
Yes maybe in later prs